django_setup_configuration/constants.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+basic_field_descriptions = {
+    models.BooleanField: "True, False",
+    models.CharField: "string",
+    models.EmailField: "string representing an Email address (foo@bar.com)",
+    models.FileField: (
+        "string representing the (absolute) path to a file, "
+        "including file extension: /absolute/path/to/file.xml"
+    ),
+    models.ImageField: (
+        "string representing the (absolute) path to an image file, "
+        "including file extension: /absolute/path/to/image.png"
+    ),
+    models.IntegerField: "string representing an integer",
+    models.JSONField: "Mapping: {example}".format(example="{'some_key': 'Some value'}"),
```

Comment:
could also be a string, float, int, None or list... :grimacing:

Reply:
Split remark off into a separate issue